themes/gatsby-starter-blog-theme/README.md

```diff
@@ -0,0 +1,95 @@
+
+    To get started, check out the guide to [getting started with using the blog theme starter](http://gatsbyjs.org/docs/themes/using-a-gatsby-theme), or the longer, [more detailed tutorial](http://gatsbyjs.org/tutorial/using-a-theme).
+
+## 🧐 What's inside?
+
+A quick look at the top-level files and directories you'll see in a site created using the blog theme starter.
+
+gatsby-starter-blog-theme
+├── content
+│ ├── assets
+│ │ └── avatar.png
+│ └── posts
+│ ├── hello-world.mdx
+│ └── my-second-post.mdx
+├── src
+│ └── gatsby-theme-blog
+│ ├── components
+│ │ └── bio-content.js
+│ └── gatsby-theme-ui
+│ └── colors.js
+├── .gitignore
+├── .prettierrc
+├── gatsby-config.js
+├── LICENSE
+├── package-lock.json
+├── package.json
+└── README.md
+
+1.  **`/content`**: A content folder holding assets that the theme expects to exist. This will vary from theme to theme -- this starter is set up to get you started with the blog theme, which expects an image asset for your avatar, and blog post content. Replace the avatar image file, delete the demo posts, and add your own!
+
+2.  **`/src`**: You will probably want to customize your site to personalize it. The files under `/src/gatsby-theme-blog` _shadow_, or override the files of the same name in the `gatsby-theme-blog` package. To learn more about this, check out the [guide to getting started with using the blog theme starter](http://gatsbyjs.org/docs/themes/using-a-gatsby-theme).
+
+3.  **`.gitignore`**: This file tells git which files it should not track / not maintain a version history for.
+
+4.  **`.prettierrc`**: This file tells git which files it should not track / not maintain a version history for.
+
+5.  **`gatsby-config.js`**: This is the main configuration file for a Gatsby site. This is where you can specify information about your site (metadata) like the site title and description, which Gatsby plugins you’d like to include, etc. When using themes, it's where you'll include the theme plugin, and any customization options the theme provides.
+
+6.  **`LICENSE`**: Gatsby is licensed under the MIT license.
+
+7.  **`package-lock.json`** (See `package.json` below, first). This is an automatically generated file based on the exact versions of your npm dependencies that were installed for your project. **(You won’t change this file directly).**
+
+8.  **`package.json`**: A manifest file for Node.js projects, which includes things like metadata (the project’s name, author, etc). This manifest is how npm knows which packages to install for your project.
+
+9.  **`README.md`**: A text file containing useful reference information about your project.
+
+## 🎓 Learning Gatsby
+
+Looking for more guidance? Full documentation for Gatsby lives [on the website](https://www.gatsbyjs.org/).
+
+Here are some places to start:
+
+### Themes
+
+- To learn more about Gatsby themes specifically, we recommend checking out the [theme docs](https://www.gatsbyjs.org/docs/themes/).
```

Comment:
```suggestion
- To learn more about Gatsby themes specifically, including composing multiple themes, check out the [theme docs](https://www.gatsbyjs.org/docs/themes/).
```

Reply:
composing multiple themes isn't a concern of this particular starter. I'm not sure I'd call that out here?

Reply:
I wondered if people who looked at these READMEs when trying to compose themes together would wonder about that. But we can skip it!